server.go

```diff
@@ -284,7 +291,16 @@ func serveVerifyAccount(c *gin.Context) {
 		c.JSON(http.StatusLocked, gin.H{"error": ErrCounterReached.Error()})
 		return
 	}
-	fiftyDataCaps := types.BigMul(env.MaxAllowanceBytes, types.NewInt(50))
+
+	maxAllowance, err := user.GetMaxAllowance(targetAddrStr)
+	if err != nil {
+		slackNotification := "CALCULATING MAX ALLOWANCE FAILED" + err.Error() + "\n----------"
+		sendSlackNotification("https://errors.glif.io/max-allowance-failed", slackNotification)
+		c.JSON(http.StatusInternalServerError, gin.H{"error": ErrMaxAllowanceFailed.Error()})
+		return
+	}
+
+	fiftyDataCaps := types.BigMul(maxAllowance, types.NewInt(50))
```

Comment:
This is basically a check to just ping me when the verifier backend gets low on datacap, i dont expect this to hit any time soon.. Anyways, for now lets just compute the max score and multiply it by 50 (warn me when we only have 50 max scores left to allocate).

So it would be something like: 
```
maxAllowanceToGrant := types.BigMul(env.BaseAllowanceBytes, types.NewInt(16))
fiftyDataCaps := types.BigMul(env.maxAllowanceToGrant, types.NewInt(50))
```

Reply:
Sounds good!